### DISCUSSION TITLE
Is this package still maintained?

### DISCUSSION THREAD

**Discussion by tyteen4a03** [ReactiveX/RxPY#717] [General]
The last release was about 3 years ago. Before I start building on top of it, I want to make sure I'm not building on top of unmaintained software.

**Comment by MainRo** [comment]
Hello, 

Yes, this package is still maintained. The last 4.x release dates from 2 years. Note that it was renamed from Rx to reactivex when we bumped from v3.x to v4.x.
We did not publish releases for a long time simply because the code is quite stable.

Personally, I still have some pending changes related to state management but I did not have time to complete this yet.

**Reply by kormang** [reply]
I thought it was abandoned too.

I made this PR, with no feedback from you: https://github.com/ReactiveX/RxPY/pull/711

First I had issues when running checks in pipeline, which were unrelated to my changes, but I fixed them anyway by updating isort.

Then I had issues with Mac build, which doesn't seem to be related to my changes, and I don't know what to do about it.

Then, after merged lastest master i have issues with formatting (black), which are again unrelated to my changes, and clearly you have merged changed that don't pass the checks, but not it looks like my PR is not passing checks.

Please, my at least some reply, at least something like "We don't care about you PR", so I can close it, instead of leaving it open for eternity.

Also, would you do something about your automated builds/checks?

**Reply by Kroppeb** [reply]
Yeah, the CI is failing on master. The fact that there is a badge saying: `Python package: failing` and that the latest release is 2 years old is making my work a bit anxious about using this package.

**Reply by cebaa** [reply]
> We did not publish releases for a long time simply because the code is quite stable.

@MainRo any reason not to publish a more-recent release? Or even set up github actions to auto-publish on new tags?

**Reply by dbrattli** [reply]
FYI: I have just did a big overhaul of the library, updated to modern tooling (uv + ruff) and have also published a new version. A new v5 is also in the making 😅
